lang/en/lang.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
 $lang['unknown'] = 'unknown cross reference';
```

Comment:
```suggestion
$lang['unknown'] = 'unknown definition';
```
'definition' explains a bit more what is searched and what not.